docs/api-ref.md

```diff
@@ -4700,21 +4700,222 @@ See [ViewItem class](#viewitem-class)
 
 
 ---
+## Webhooks
+<br>
+<br>
+Using the Tableau Server Client (TSC), you can create a new webhook, get a list of all the webhooks, get details about a specific webhook, or delete a webhook. 
+<br>
 
+The webhook resource for Tableau Server and Tableau Cloud are defined in the `WebhookItem` class. The class corresponds to the webhook resources you can access using the Tableau REST API.  For example, using REST API, you can gather information about a workbook, like the name of the Webbook, the event it is associated with, and the destination URL, and you can get the same information using TSC as well.
+<br>
 
-## Workbooks
+Tableau webhook REST API endpoints are available in **REST API version 3.16** and later. 
+<br>
+<br>
 
-Using the TSC library, you can get information about a specific workbook or all the workbooks on a site, and  you can publish, update, or delete workbooks.
+### WebhookItem class
+The `WebhookItem` represents the webhook resources on Tableau Server or Tableau Cloud. This is the information that can be sent or returned in response to a REST API request for webhooks.  
 
-The project resources for Tableau are defined in the `WorkbookItem` class. The class corresponds to the workbook resources you can access using the Tableau REST API. The workbook methods are based upon the endpoints for projects in the REST API and operate on the `WorkbookItem` class.
+**Attributes**
 
+Name | Description
+:--- | :---
+`id` |  The identifier (*luid*) for the webhook. You need this value to query a specific webhook  with the `get_by_id` method or to delete a webhook with the  `delete` method.
+`name`  |  The name of the webhook. You must specify this when you create an instance of the `WebhookItem`.
+`url` | The destination URL for the webhook. The webhook destination URL must be https and have a valid certificate. You must specify this when you create an instance of the `WebhookItem`.
+`event` | The name of the Tableau event that triggers your webhook.This  is either `api-event-name` or `webhook-source-api-event-name`: one of these is required to create an instance of the `WebhookItem`. We recommend using the `api-event-name`. <br> The event name must be one of the supported events listed in the [Trigger Events](https://help.tableau.com/current/developer/webhooks/en-us/docs/webhooks-events-payload.html) table. 
```

Comment:
@jacalata I added a recommendation here to use the short name.  Let me know if we should actually just remove the 'webhook-source-api-event-name'.

Reply:
No, it's still in the code and some people are already using it, so it's better to have it documented.